src/components/Figure.tsx

```diff
@@ -1,24 +1,35 @@
 import { FunctionComponent, ReactNode } from 'react'
 
 import classNames from 'classnames'
-
+import OpenInNewIcon from 'mdi-react/OpenInNewIcon'
```

Comment:
The react-icons package mentioend in #5743 has some [external link icons](https://react-icons.github.io/react-icons/search?q=externallink).

```suggestion
import { FiExternalLink } from 'react-icons/fi'

```

Reply:
Sweet! Would we want to import feathers' version of this icon instead of material design because `react-icons/fi` is smaller than `react-icons/md`? I'll wait for #5743 to merge & can add this in 😄

Reply:
Yes for sure! Let's not block this in the mean time!

I asked Tracey to look at `react-icons` so we'll wait to hear back to see what would work well with branding. 😁